test/browser/spec/ol/graticule.test.js

```diff
@@ -187,8 +187,6 @@ describe('ol.layer.Graticule', function () {
       graticule.createGraticule_(extent, [0, 0], resolution, squaredTolerance);
       expect(graticule.meridiansLabels_[0].text).to.be('lon: 0');
       expect(graticule.parallelsLabels_[0].text).to.be('lat: 0');
-      expect(graticule.lonLabelStyle_(feature).getText()).to.eql(lonLabelStyle);
-      expect(graticule.latLabelStyle_(feature).getText()).to.eql(latLabelStyle);
```

Comment:
This was asserting that cloned features shared style instances.  This is no longer the case.  A cloned feature will get a cloned style.